### PR TITLE
修复Linux版本严重错误，细节微调，更正大量语法错误，添加中文README。

### PR DESCRIPTION
**Pull request by Mivik**
RT

~~（话说你有在 Linux 上测试吗... 你原来的版本多加了一个/根本不能用来着（（~~

### CODE REVIEW

**Comment by xkeyC** [comment]
坚持 bin 原则，另外不建议使用复制，后期更新可通过 git pull获取

File: scripts/fgit.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Check arguments
+if [ "$1" == "" ];then 
+    echo Error: Expected a GitHub repository URL after fgit.
+    exit 1
+fi
+
+# Check URL
+if [[ "$1" != *"github.com"* ]]
+then
+echo "Error: $1 is not a GitHub URL!"
```
Comment (KevinZonda) [inline scripts/fgit.sh]: indentation

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# fgit
+一个在克隆项目时自动把 GitHub 链接替换为 fastgit 链接的工具。
+
+[English](./README.md)
+
+## 如何使用？
```
Comment (KevinZonda) [inline README-zh_CN.md:7]: MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines 
See more [here](https://github.com/updownpress/markdown-lint/blob/master/rules/022-blanks-around-headers.md).

File: README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 # fgit
-A tool to replace your github URL to fastgit URL
+A tool to replace your GitHub URL with fastgit URL automatically when cloning repositories.
 
-## How To Use？
+[简体中文](./README-zh_CN.md)
+
+## How To Use?
```
Comment (KevinZonda) [inline README.md:7]: MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines 
See more [here](https://github.com/updownpress/markdown-lint/blob/master/rules/022-blanks-around-headers.md).

File: README.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 ### Linux
```
Comment (KevinZonda) [inline README.md:9]: Maybe adding Unix is better?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
 1. Clone fgit
-2. eq: `sudo ln -s /home/username/tools/fgit/bin/fgit.sh /bin/fgit`
```
Comment (KevinZonda) [inline README.md:7]: Using `~` to replace `/home/username/`

File: README.md [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
-3. Use commands `fgit https://github.com/xxx/xxx.git ` to  use the fastgit.org clone repository
+2. Execute in fgit folder: `sudo cp ./scripts/fgit.sh /usr/local/bin`
+3. Use command `fgit https://github.com/xxx/xxx.git` to  use fastgit.org to clone the repository
```
Comment (KevinZonda) [inline README.md]: Two spaces inserted before `use fastgit,org` should be removed.

File: scripts/fgit.sh [hunk 7]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# author: xkeyC
```
Comment (KevinZonda) [inline scripts/fgit.sh]: Add your name

File: scripts/fgit.sh [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+# url: github.com/xkeyC
```
Comment (KevinZonda) [inline scripts/fgit.sh]: Add your github url

**Review by KevinZonda — CHANGES_REQUESTED** [review]

File: scripts/fgit.sh [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+# Check arguments
+if [ "$1" == "" ];then
```
Comment (KevinZonda) [inline scripts/fgit.sh]: Uniform code style. Use 
```bash
if [ ]; then
fi
```

or 

```bash
if [ ]
then
fi
```

**Review by KevinZonda — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Mivik) [reply README.md:7]: it's removed

File: scripts/fgit.sh (hunk 9 above)
Comment (Mivik) [reply scripts/fgit.sh]: note that i didn't modify the style.

**Review by KevinZonda — APPROVED** [review]

**Pull request merged**